taxi/tests/test_forms.py

```diff
@@ -0,0 +1,38 @@
+from django.test import TestCase
+
+from taxi.forms import DriverCreationForm, DriverSearchForm
+
+
+class FormsTests(TestCase):
+    def test_driver_creation_form_license_first_last_name_is_valid(self):
+        form_data = {
+            "username": "new_test_username",
+            "password1": "user13test",
+            "password2": "user13test",
+            "license_number": "ADT12345",
+            "first_name": "first_name",
+            "last_name": "last_name",
+        }
+        form = DriverCreationForm(data=form_data)
+        print(form.errors)
+        self.assertTrue(form.is_valid())
+        self.assertEqual(form.cleaned_data, form_data)
```

Comment:
The assertion `self.assertEqual(form.cleaned_data, form_data)` may fail because `cleaned_data` typically contains additional fields or transformations not present in the initial `form_data`. Consider checking individual fields or using `assertDictContainsSubset` for a more reliable test.